src/components/Buttons/ButtonLink.tsx

```diff
@@ -4,16 +4,26 @@ import { BaseLink, type LinkProps } from "@/components/Link"
 import { type MatomoEventOptions, trackCustomEvent } from "@/lib/utils/matomo"
 
 export type ButtonLinkProps = LinkProps &
-  Omit<ButtonProps, "toId" | "onClick"> & {
+  Omit<ButtonProps, "toId"> & {
     customEventOptions?: MatomoEventOptions
   }
 
-const ButtonLink = ({ customEventOptions, ...props }: ButtonLinkProps) => {
+const ButtonLink = ({
+  customEventOptions,
+  onClick,
+  ...props
```

Comment:
Hm.. this is a little confusing since we're accepting an `onClick`, which could override the click handler per below, but we also have `customEventOptions` which wouldn't be used if `onClick` is provided... 

@nhsz If we need to accept `onClick` here, should we merge it into the `handleClick()` function below? ie

```tsx
const handleClick = () => {
  customEventOptions && trackCustomEvent(customEventOptions)
  onClick()
}
```

Reply:
@nhsz @wackerow the underlying [`Button` component already handles this logic](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/Buttons/Button.tsx#L43). So I'd say, let's get rid of the `onClick` handler from this component completely and rely on the logic set in the button component. Otherwise, we are (and we were) duplicating this logic.